README.md

```diff
@@ -269,7 +269,8 @@ this.alfrescoApi = new AlfrescoApi({
             implicitFlow: true,
             redirectUri: 'YOUR_HOME_APP_URL',
             silentRefreshTimeout: '600000' //Optional parameter 10 minutes default value,
-            silentLogin: true
+            silentLogin: true,
+            silentLoginExceptList: ['{protocol}//{hostname}{:port}/#/preview/s']
```

Comment:
Do you think is better silentLoginExceptList or silentLogiWhiteList

Reply:
"publicRoutes", "publicUrls"....

I think "silent login exception / whitelist" makes sense just for us that know the entire story, but for others might not mean anything at a first glance

Reply:
I think "silentLogin" implies the knowledge of what the silent means in the authentication, using simpler terms like "publicUrls" may raise less questions as everyone knows what public in URLs mean